var/spack/repos/builtin/packages/aluminum/package.py

```diff
@@ -50,7 +50,12 @@ class Aluminum(CMakePackage, CudaPackage, ROCmPackage):
         description="Builds with support for CUDA intra-node "
         " Put/Get and IPC RMA functionality",
     )
-    variant("rccl", default=False, description="Builds with support for NCCL communication lib")
+    variant("rccl", default=False, description="Builds with support for RCCL communication lib")
+    variant(
+        "libfabric",
+        default=False,
```

Comment:
```suggestion
        default=False,
        when='+rccl',
```
Then the new conflict is unnecessary

Reply:
@becker33 We need to be careful because `libfabric` support is only valid on some `cray` systems, but not all.